Add Current Date and Time for reference

dracosplace opened the pull request:
Don't have a dev environment currently so taking a pass at this as an example vs. a non-code based issue. 

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Breaking change
<!--
  If your PR contains a breaking change for existing users, it is important
  to tell them what breaks, how to make it work again and why we did this.
  This piece of text is published with the release notes, so it helps if you
  write it towards our users, not us.
  Note: Remove this section if this PR is NOT a breaking change.
-->


## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue in the
  additional information section.
-->
Would be great to have the systems current date / time somewhere appropriate.
Somewhere on the "Supervisor" -> "System" tab, or "Configuration" -> "Info" tab would be great

## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [X] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [X] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [ ] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [ ] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [ ] Untested files have been added to `.coveragerc`.

The integration reached or maintains the following [Integration Quality Scale][quality-scale]:
<!--
  The Integration Quality Scale scores an integration on the code quality
  and user experience. Each level of the quality scale consists of a list
  of requirements. We highly recommend getting your integration scored!
-->

- [ ] No score or internal
- [ ] 🥈 Silver
- [ ] 🥇 Gold
- [ ] 🏆 Platinum

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html
[quality-scale]: https://developers.home-assistant.io/docs/en/next/integration_quality_scale_index.html
[docs-repository]: https://github.com/home-assistant/home-assistant.io

dracosplace committed:
Add Current Date and Time for reference

Don't have a dev environment currently so taking a pass at this as an example vs. a non-code based issue. 

Would be great to have the time somewhere appropriate on the "Supervisor" -> "System" tab, or "Configuration" -> "Info" tab

homeassistant commented:
Hi @dracosplace, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;core#41694).

Once you do that we will be able to review and accept this pull request.

Thanks!

dracosplace committed:
fix isort error

javicalle commented on homeassistant/helpers/system_info.py:
```diff
@@ -9,6 +10,8 @@
 
 from .typing import HomeAssistantType
 
+DATE_STR_FORMAT = "%Y-%m-%d %H:%M:%S"
```
I really do not know where it can be obtained in HA but, shouldn't the timeformat of the user's locale be used here?

dracosplace replied:
Good call out. Looks like %c might be what we could use here without pulling in Babel or some other library.

%c : Locale’s appropriate date and time representation. : 
Examples:
Tue Aug 16 21:30:00 1988 (en_US);
Di 16 Aug 21:30:00 1988 (de_DE)

dracosplace committed:
Format date / time based on Locale

Switching this from a pre-formatted date time format string to the built in locale aware format string. 

Details from the python datetime docs:
%c : Locale’s appropriate date and time representation.

balloob commented:
You can't add some value without a plan to add this to the frontend or test it locally with a dev env. This seems more like a feature request, which should go in the forums.

dracosplace commented:
> You can't add some value without a plan to add this to the frontend or test it locally with a dev env. This seems more like a feature request, which should go in the forums.

@balloob appreciate the feedback. 

From reading the frontend code, I added it here so that it would get iterated over and just added to the list of Systems info on the systems tab without requiring a frontend change. My apologies if I'm misinterpreting / misreading the frontend logic.

I was hoping that the reviewer would have a dev environment already setup that they could quickly validate this feature request against due to the size of the change compared to the effort to setup a dev environment. It's 100% a feature request but I was trying to make things simpler by providing a potential/example solution vs. just asking for a feature. Seems like that was not desired / helpful in this case.
